site/docs/v1/tech/apis/_user-search-request-parameters-elasticsearch.adoc

```diff
@@ -0,0 +1,38 @@
+[api]
+
+[field]#search.query# [type]#[String]# [optional]#Optional# [since]#Available since 1.13.0#::
+The raw JSON Elasticsearch query that is used to search for Users.  The [field]#ids#, [field]#query#, and [field]#queryString#  parameters are mutually exclusive, they are listed here in order of precedence.
++
+It is necessary to use the [field]#search.query# parameter when querying against `registrations` in order to achieve expected results, as this field is defined as a link:https://www.elastic.co/guide/en/elasticsearch/reference/6.3/nested.html[nested datatype] in the Elasticsearch mapping.
++
+[field]#search.queryString# [type]#[String]# [optional]#Optional#::
+The Elasticsearch query string that is used to search for Users.  The [field]#ids#, [field]#query#, and [field]#queryString#  parameters are mutually exclusive, they are listed here in order of precedence.
++
+[field]#search.sortFields[``x``].missing# [type]#[String]# [optional]#Optional# [default]#defaults to `_last`#::
+The value to substitute if this field is not defined. Two special values may be used:
++
+* `_first` When the field is not defined sort this record first.
+* `_last` When the field is not defined sort this record last.
++
+
+[field]#search.sortFields[``x``].name# [type]#[String]# [optional]#Optional#::
```

Comment:
This really isn't an optional field. When sort fields are provided, the name is required. 

Sort fields themselves are optional, so if you see `[field]#sortFields# [type]#[Array]# [optional]#Optional#::` is optional, because you don't have to provide any sort fields. 

But once you do, the `name` parameter is required. This is the pattern we have across the API docs. 

Maybe there is a better way to communicate this to the user.

Reply:
Ah, ok. I'll revert. Definitely want to stay consistent across the other API docs. 

I was a bit confused because `required` to me means required in all uses. Maybe something like a `dependent on <field>` tag. Will mull it over a bit more.